### PR TITLE
Update README.md to reflect ability to use with WDIO v7

### PR DESCRIPTION
**Pull request by mikesalvia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ wdio-video-reporter [![Build Status](https://travis-ci.org/presidenten/wdio-vide
 
 ![Logo](https://raw.githubusercontent.com/presidenten/wdio-video-reporter-example-report/master/wdio-video-reporter.png)
 
-This is a reporter for [Webdriver IO v6](https://webdriver.io/) that generates videos of your wdio test executions. If you use allure, then the test cases automatically get decorated with the videos as well. (For Webdriver IO v5, please use wdio-video-reporter version ^2.0.0.)
+This is a reporter for [Webdriver IO v6+](https://webdriver.io/) that generates videos of your wdio test executions. If you use allure, then the test cases automatically get decorated with the videos as well. (For Webdriver IO v5, please use wdio-video-reporter version ^2.0.0.)
```
Comment (christian-bromann) [inline README.md:6]: How about being more explicit?
```suggestion
This is a reporter for [Webdriver IO v6 and highr](https://webdriver.io/) that generates videos of your wdio test executions. If you use allure, then the test cases automatically get decorated with the videos as well. (For Webdriver IO v5, please use wdio-video-reporter version ^2.0.0.)
```

**Comment by mikesalvia** [comment]
@christian-bromann I am going to close PR, I actually have replicated this as a service that plays nicely with other reporting services. Let me know if you want to keep this open I will leave it but it has been a number of days.

**Comment by christian-bromann** [comment]
> I actually have replicated this as a service that plays nicely with other reporting services

Can you elaborate? Is this something you can contribute to the ecosystem?

**Comment by christian-bromann** [comment]
Will go ahead and merge. @mikesalvia please let me know if you have any upstream changes that can be merged into this package. Thanks!

**Pull request merged**